src/themes/deprecated/foundational-styles/index.js

```diff
@@ -0,0 +1,5 @@
+export { default as colors } from './colors'
+export { default as fills } from './fills'
+export { default as palette } from './palette'
+export { default as tokens } from './tokens'
```

Comment:
I don't think `tokens` was a thing previously.

Reply:
This isn't even a valid file on disk 😅 , but build wasn't complaining because we don't re-import it anywhere